controller_app/views.py

```diff
@@ -1,8 +1,42 @@
-from django.http import JsonResponse
+import logging
+
+from django.http import HttpResponse, JsonResponse
 from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+
+from jobrunner.controller.task_api import get_active_tasks, handle_task_update
+
+
+log = logging.getLogger(__name__)
 
 
 @csrf_exempt
 def index(request):
     response = JsonResponse({"method": request.method})
     return response
+
+
+def active_tasks(request, backend):
+    active_tasks = [task.asdict() for task in get_active_tasks(backend)]
+    return JsonResponse({"tasks": active_tasks})
```

Comment:
I think we should be serializing AgentTask's here, not db Tasks?

The point of AgentTask was a simplified serializable task definition with just the data the agent needs, not the db level stuff?

Reply:
The conversion of db Task -> AgentTask that happens in the current agent.task_api is an implementation detail of the fake db api, not a deliberate step that needs to happen on the agent side.